File: 백준/Bronze/10807. 개수 세기/개수 세기.c
#include <stdio.h>
int main(){
  int n;
  int a[100];
  scanf("%d\n",&n);

  for(int j=0; j<n; j++){
    scanf("%d",&a[j]);
  }

  int v;

  scanf("%d",&v);
  
  int count =0;
  
  for(int i=0; i<n; i++){
    if(v == a[i])
      count++;
  }
  printf("%d",count);
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String args[]) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int num = Integer.parseInt(br.readLine());
        
        StringTokenizer st = new StringTokenizer(br.readLine());
        int[] arr = new int[num];
        
        for(int i=0; i<arr.length; i++){
            arr[i] = Integer.parseInt(st.nextToken());
        }
        
        int searchNum = Integer.parseInt(br.readLine());
        int count = 0;
        
        for(int i=0; i<arr.length; i++){
            if(arr[i] == searchNum){
                count++;
            }
        }
        System.out.println(count);
    }
}